modules/tinymce/src/plugins/table/test/ts/browser/TabKeyNavigationTest.ts

```diff
@@ -74,4 +74,114 @@ describe('browser.tinymce.plugins.table.TabKeyNavigationTest', () => {
     assert.equal(events[1].type, 'tablemodified');
     editor.off('TableModified', logEvent);
   });
+
+  context('TINY-6683: Tab navigation with ranged selection', () => {
```

Comment:
We haven't previously included JIRA numbers in contexts, as it can contain multiple tests with different JIRA numbers, so maybe this instead?
```suggestion
  context('Tab navigation with ranged selection', () => {
```